README.md

```diff
@@ -169,10 +170,63 @@ ATTR{suspend_mode}=="[oem|keep|off]"
 ```
 
 As en example, to set `suspend_mode`to `off`:
-`ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
+`ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"
 `
 
 to create the rule in one line:
 ```shell
-$ echo 'ACTION=="add|change", KERNEL=="multicolor:chassis", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
+$ echo 'ACTION=="add|change", KERNEL=="ayaneo:rgb:joystick_rings", SUBSYSTEM=="leds", ATTR{suspend_mode}="off"' | sudo tee /etc/udev/rules.d/99-led_suspend_mode.rules
 ```
+
+### Bypass Charge Control
+
+Bypass Charge can be enabled, disabled, start and stop battery level can be set via sysfs.
+
+n most systems this will be mounted at `/sys/class/power_supply/ayaneo:bypass_charge/` and provides the following files:
+
+#### charge_type
+
+Read/write.
+
+Enables or disables the bypass charge. Accepts a value of 6 and 0 or "Custom" and "Unknown".
+
+|Value|Description|
+|-|-|
+|6 or "Custom"|Enables the bypass charge. Battery charging will depend on the start and stop values.|
+|0 or "Unknown"|Disables the bypass charge. The battery is normally charged.|
+
+Default is "Custom"
+
+#### charge_control_start_threshold
+
+Read/write.
+
+Gets or sets the battery percentage level, below which charging will begin. Valid values between 0 and 100 (percent).
+
+Default is 85.
```

Comment:
Again, we should default to normal behavior. We don't want to generate bug reports saying Linux can only charge my device to 85%.